docs/source/commands.rst

```diff
@@ -178,7 +178,36 @@ optional arguments:
 
 ::
 
-    $cfncluster ssh mycluster -i ~/.ssh/id_rsa -v
+    $ cfncluster ssh mycluster -i ~/.ssh/id_rsa -v
+
+stepfunctions
+=============
+
+Creates a step function that automatically creates a cluster, runs user specified jobs, and tears the cluster down.
+
+For example:
+    cfncluster stepfunctions -b s3bucket -j path/to/jobs.config
+
+This uses the bucket name s3bucket, or creates it if it doesn't exist, to store the lambda source code and user specified jobs. To see how jobs in the jobs.config file should be specified, see // TODO :)

```

Comment:
Remove TODO and link page

Reply:
Also, you can add \`\` for `jobs.config` , see https://github.com/awslabs/cfncluster/blame/develop/docs/source/configuration.rst#L5